### DISCUSSION TITLE
Stack with id <stack> does not exist

### DISCUSSION THREAD

**Discussion by pawanreddy1708** [serverless/serverless#11732] [Q&A] [answered]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest v3 release?

- [X] Yes, I'm using the latest v3 release

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

Hi Team,

I am fairly new to serverless, I have observed when i try to deploy the serverless.yml file it gives me the following error as mentioned in the title, I read through other similar issue, region us-east-2 which is mentioned under the region section, really not sure where it is failing as it does not give any error

I got this error when i ran -> serverless info

### Service configuration (serverless.yml) content

```yaml
N/A
```

### Command name and used flags

serverless info

### Command output

```shell
node_modules/serverless/bin/serverless.js info
Environment: darwin, node 18.13.0, framework 3.27.0 (local), plugin 6.2.3, SDK 4.3.2
Credentials: Local, "default" profile
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
Stack with id okta-lambda-domain-manager-service-pawreddy-dev does not exist
```

### Environment information

```shell
Framework Core: 3.27.0 (local)
Plugin: 6.2.3
SDK: 4.3.2
```

**Comment by medikoo** [comment, marked as answer]
@pawanreddy1708 `sls info` works only for already deployed services

**Comment by pawanreddy1708** [comment]
@medikoo thanks for that lightning speed response

Oh, my bad, but even if i try to deploy it for the first time with the command -> serverless deploy, it could not deploy and gave me the below error.



Deploying okta-lambda-domain-manager-service-pawreddy to stage test (us-east-2)

✖ Stack okta-lambda-domain-manager-service-pawreddy-dev failed to deploy (0s)
Environment: darwin, node 18.13.0, framework 3.27.0 (local), plugin 6.2.3, SDK 4.3.2
Credentials: Local, "default" profile
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
Error: spawn python3.8 ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:476:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

**Comment by medikoo** [comment]
@pawanreddy1708 yes, error states that you miss the Python runtime.

I'm moving it to discussions as it's more a support request and not a bug report
